app/controllers/subscriptionController.ts

```diff
@@ -49,13 +49,25 @@ export const broadcast = async (
     const subscriptions = await subscriptionRepository.getAll();
 
     const notifications: Promise<SendResult>[] = [];
+
+
+    
+
     subscriptions.forEach((subscription) => {
-      notifications.push(webpush.sendNotification(subscription, JSON.stringify(notification)));
+      console.log(`sub:${subscription}`);
+      const options = {
+            proxy: 'http://localhost:7890',
```

Comment:
Why did you add this? Will this work for others as well?

Reply:
it is a example for people need a proxy :)